src/stream.rs

```diff
@@ -50,7 +50,7 @@ impl Default for OutputStreamConfig {
             channel_count: 2,
             sample_rate: HZ_44100,
             buffer_size: BufferSize::Default,
-            sample_format: SampleFormat::I8,
```

Comment:
why not f32? If its supported by the OS that will probably have the lowest overhead.

Reply:
Just forgot to update. But maybe I16 can be a better default since it is widely supported.